internal/pkg/attestationpolicy/attestationpolicy.go

```diff
@@ -160,3 +153,15 @@ func getAPDNSNameTemplatesHelmConfig(dnsNameTemplates []string) []string {
 	}
 	return dnsNameTemplates
 }
+
+func renderSPIFFEID(trustDomain, path string) (string, error) {
+	td, err := spiffeid.TrustDomainFromString(trustDomain)
+	if err != nil {
+		return "", err
+	}
+	spiffeID, err := spiffeid.FromPath(td, "/"+path)
```

Comment:
Is this consistent with the template defined at `attestationpolicy.go` with the trailing slash in the trust domain?

```
// spiffeTDIDTemplate is a Go template for a SPIFFE trust domain.
const spiffeTDIDTemplate = "spiffe://{{ .TrustDomain }}/"
```

Reply:
Yes. The k8s attestation policy spiffe id path template should not have a leading slash, as we add one in the above template. The spiffeid package requires a leading slash in the path, so we add one here. If a user did specify a leading slash it would currently fail with an empty path component error (`//`)

I suppose there's a question of whether we want to allow, require or reject a leading slash in our paths. Perhaps we should be agnostic?